### DISCUSSION TITLE
C:\ThisExe\WasNotFound\InPath\dotnet-tools-outdated.exe'

### DISCUSSION THREAD

**Discussion by dcog989** [Devolutions/UniGetUI#1861] [Q&A] [answered]
I don't think this is a bug so asking for advice here:

```
Starting update check
Downloaded icons and screenshots successfully!
WingetUI is up to date
Api running on http://localhost:7058
Lang files were updated successfully
LoadComponentsAsync finished executing. All managers loaded. Proceeding to interface.
Error finding updates on manager .NET Tool: 
System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\ThisExe\WasNotFound\InPath\dotnet-tools-outdated.exe' with working directory 'C:\Program Files\WingetUI'. The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at ModernWindow.PackageEngine.Managers.Dotnet.GetAvailableUpdates_UnSafe() in Y:\WingetUI-Store\src\wingetui\PackageEngine\Managers\Dotnet.cs:line 82
   at ModernWindow.PackageEngine.Classes.PackageManager.GetAvailableUpdates() in Y:\WingetUI-Store\src\wingetui\PackageEngine\Classes\Manager.cs:line 128
Invalid value for UpdatesCheckInterval, using default value of 3600 seconds
```

I've reinstalled .NET SDK, .NET Desktop Runtime, and WingetUI but no luck. 

I think this is also affecting winget - it doesn't appear to be updating its catalog.

I'm running - https://github.com/microsoft/winget-cli/releases/tag/v1.7.10582

Any suggestions please?

**Comment by marticliment** [comment, marked as answer]
Are you using the dotnet tool package manager?
If so, please try running `dotnet tool install --global dotnet-tools-outdated` and restart WingetUI

**Comment by dcog989** [comment]
> Are you using the dotnet tool package manager? If so, please try running ...

Not that I'm aware of, although I have been playing around in VS Code and did install '.NET Install Tool' extension - but can't remember when.

However, running that command has fixed the problem. Just the warning:

`Invalid value for UpdatesCheckInterval, using default value of 3600 seconds`

Thanks (as always!) for your help. :)

**Reply by marticliment** [reply]
I see. The warning you are seeing is expected if you haven't modified that setting, so just ignore it. 

The command that I shared with you installs a required dependency that should have been installed by default, 
I am going to investigate this

**Comment by DrStrange** [comment]
My packages stopped appearing and WingetUI showed this error. However, after installing the .NET SDK to run the `dotnet tool install --global dotnet-tools-outdated` command, my packages did not reappear after restarting WingetUI. I then had to follow the steps stated in  #1721 before my packages appeared. I did try the steps outlined in #1721 first before installing the .NET SDK. @Marti, after you investigate and add the required dependency to fix this error, could you mention it in the release notes, so I can uninstall the .NET SDK, as I don't need it. Thanks.
